tensorrt_llm/_torch/attention_backend/trtllm.py

```
@@ -1050,48 +1050,49 @@ def prepare_context_mla_with_cached_kv(self,
+        spec_metadata,
+        spec_tree_manager,
+        max_draft_len,
+        max_total_draft_tokens,
         spec_decoding_tensor: Optional['SpecDecodingTensor'] = None,
     ):
-
         if spec_decoding_tensor is not None:
-            spec_decoding_position_offsets = spec_decoding_tensor.position_offsets
-            spec_decoding_packed_mask = spec_decoding_tensor.packed_mask
-            spec_decoding_generation_lengths = spec_decoding_tensor.generation_lengths
+            spec_decoding_tensor.position_offsets
```

Ruff (0.14.1)

Found useless expression. Either assign it to a variable or remove it.

(B018)